lib/question.js

```diff
@@ -1337,6 +1408,32 @@ module.exports = {
                     callback(null);
                 });
             },
+            (callback) => {
+                // if (locals.submissions[0].submitted_answer['_file_upload_storage']) {
+                //     this._subReferencesWithFiles(locals.submissions[0]);
+                // }
+
+                if (locals.submissions[0] && locals.submissions[0].submitted_answer['_file_upload_storage']) {
+                    var params = {
+                        file_ids: locals.submissions[0].submitted_answer['_file_upload_storage']['file_ids'],
+                    };
+
+                    sqldb.query(sql.select_files, params, (err, result) => {
+                        if (ERR(err, callback)) return;
+                        if (result.rowCount >= 1) {
+                            // Do I want to fetch the file through S3 or send hyperlinks back?
```

Comment:
Is there an easy way to assemble the hyperlink to the API in *.js files? I have not found one yet. We have the assessment_instance_id, question_instance, etc. to form any hyperlink on the back-end, but we may not have that on the front-end.

I will figure out this logic soon, but open to input!